examples/runcards/report.md

```diff
@@ -5,15 +5,18 @@
 {@table_two_point_scalars@}
 ## Two Point Correlator
 {@plot_two_point_correlator@}
+{@plot_two_point_correlator_error@}
```

Comment:
do we want to keep adding to this report, or at some point provide some more examples of individual actions or smaller reports?

Reply:
Yeah I have opened an issue which is almost the same as addressing this #78